openid-federation-1_0.xml

```diff
@@ -6057,17 +5982,28 @@ GET /authorize?
             <t>
               <xref target="RFC9126">Pushed Authorization Requests</xref>
               provide an
-              interoperable way to push a Request Object
-              directly to the AS in exchange for a
+              interoperable way to push an authorization request
+              directly to the AS in exchange for a one-time-use
               <spanx style="verb">request_uri</spanx>.
+	      The standard PAR metadata parameters are used in the
+	      RP and OP metadata to indicate its use.
             </t>
+	    <t>
+	      When used with OpenID Federation,
+	      a Request Object MUST be sent to the PAR endpoint,
```

Comment:
Request objects were not required for the PAR endpoint in Federation. Federation clients authenticate at the PAR endpoint as they would at the token endpoint, with `private_key_jwt` or `self_signed_tls_client_auth`.

In the PAR RFC request objects are optional and we should keep that in Federation.

> The endpoint MAY also support sending the set of authorization request parameters as a Request Object according to [[RFC9101](https://datatracker.ietf.org/doc/html/rfc9101)] and [Section 3](https://datatracker.ietf.org/doc/html/rfc9126#request_parameter) of this document.